utils/lib/connection_database.sql

```diff
@@ -252,3 +262,42 @@ CREATE TABLE constant_sources(
     FOREIGN KEY(vcc_track_pkey) REFERENCES track(pkey),
     FOREIGN KEY(gnd_track_pkey) REFERENCES track(pkey)
 );
+
+-- Grid location map.
+CREATE TABLE grid_loc_map(
+    phy_tile_pkey INT,
+    vpr_tile_pkey INT,
+    FOREIGN KEY(phy_tile_pkey) REFERENCES phy_tile(pkey),
+    FOREIGN KEY(vpr_tile_pkey) REFERENCES tile(pkey)
+);
+
+-- Tile type map.
+-- Maps physical tiles to VPR tiles. Used for CLB splitting
+CREATE TABLE tile_type_map(
+    phy_tile_type_pkey INT,
+    vpr_tile_type_pkey INT,
+    FOREIGN KEY(phy_tile_type_pkey) REFERENCES tile_type(pkey),
+    FOREIGN KEY(vpr_tile_type_pkey) REFERENCES tile_type(pkey)
+);
+
+-- Former site table
+-- Contains entries only for CLB split products. Holds information
+-- which site instance correspond to the new tile. It is necessary to know
+-- that when remapping routing resources.
+CREATE TABLE tile_former_site(
+    vpr_tile_pkey INT,
+    site_pkey INT,
+    FOREIGN KEY(vpr_tile_pkey) REFERENCES tile(pkey),
+    FOREIGN KEY(site_pkey) REFERENCES site(pkey)
+);
+
+-- Pip in tile instance
+-- Relates a pip_in_tile with a concrete tile instance in the VPR grid.
+-- This is necessary as in the VPR we have generic SLICE tiles but pips
+-- inside them are not generic.
+CREATE TABLE pip_in_tile_instance(
```

Comment:
Just call this the "pip" table, just like there is a "wire" table and a "pip_in_tile" table.

Reply:
Ok. But mind that this table does not contain all pips. Just those for the tiles that are split.

Reply:
It needs to contain all pips, otherwise the lookup becomes inconsistent.